web/packages/teleport/src/components/hooks/useUrlFiltering/useUrlFiltering.ts

```diff
@@ -66,12 +68,13 @@ export function useUrlFiltering(
     return {
       ...initialParamsState,
       ...urlParams,
+      includedResourceMode,
       pinnedOnly:
         urlParams.pinnedOnly !== undefined
           ? urlParams.pinnedOnly
           : initialParamsState.pinnedOnly,
     };
-  }, [search]);
+  }, [search, includedResourceMode]);
 
   function setParams(newParams: ResourceFilter) {
```

Comment:
I feel like it'd be nice to omit `includedResourceMode` from here as well, perhaps create a new type in this file that can be reused? Something like `type URLResourceFilter = Omit<ResourceFilter, 'includedResourceMode'>`.

Reply:
This would avoid a situation where someone passes `includedResourceMode` to `setParams` since it's allowed by types but doesn't actually do anything.